content/sensu-core/1.4/platforms/sensu-on-mac-os-x.md

```diff
@@ -187,7 +187,7 @@ $ sudo -u _sensu /opt/sensu/bin/sensu-client -V
 
 [1]:  https://sensuapp.org/download
 [2]:  https://sensu.io/features/downloads
-[3]:  https://sensu.io/features/downloadssensu-1.4.1-1.mountainlion.pkg
+[3]:  https://sensu.io/features/downloads
```

Comment:
This will make line 189 and 190 the same. Should we just re-use 189 or de-duplicate this later?

Reply:
This leads to the question of what versions of OS X we currently support. Now that the old link to the Mountain Lion package (https://sensu.global.ssl.fastly.net/osx/sensu-1.2.0-1.mountainlion.pkg) redirects to the main site, should we remove the note about supporting it entirely?

Reply:
Talked to cwj about this. I'll replace this link with http://repositories.sensuapp.org/osx/